HomeWork1/main.swift

```diff
@@ -7,5 +7,90 @@
 
 import Foundation
 
-print("Hello, World!")
+// Задание 1: Напишите переменные и константы всех базовых типов данных: int, UInt, float, double, string. У чисел вывести их минимальные и максимальные значения.
+
+var temperature: Int = -10
+
+let floor: Int = 5
+
+var age: UInt = 10
+
+let growth: UInt = 165
+
+var cost1: Float = 100.5
+
+let cost2: Float = 55.55
+
+var number1: Double = 157.69486968
+
+let number2: Double = -837.78968995
+
+var shop: String = "Lego"
+
+let name: String = "Tom"
+
+//max и min значения
+
+let minInt = Int.min
+
+let maxInt = Int.max
+
+let Unit = UInt.max
```

Comment:
у нас камел кейс(написание переменных начинается с маленькой буквы, и каждое следующее с заглавной)
С большой буквы начинаются только объекты(классы, энамы и структуры ну и протоколы, это будем проходить дальше)
```suggestion
let uIntMax = UInt.max
```